robot/resources/lib/python_keywords/acl.py

```diff
@@ -246,10 +254,12 @@ def eacl_rules(access: str, verbs: list, user: str) -> list[str]:
     return rules
 
 
-def sign_bearer(shell: Shell, wallet_path: str, eacl_rules_file: str) -> None:
+def sign_bearer(
+    shell: Shell, wallet_path: str, eacl_rules_file_from: str, eacl_rules_file_to: str, json: bool
```

Comment:
Do we have `json=False` anywhere? If not - why change it now?

Reply:
Please look at the `bearer_token_no_limit_for_others` metod, in case of http we need a new file where bearer token is represented in binary format.